Added example and documentation for fips compliant communication with Kafka cluster (#1582)
confluentinc/confluent-kafka-python: CHANGELOG.md

# Confluent's Python client for Apache Kafka

## v2.2.0

v2.2.0 is a feature release with the following features, fixes and enhancements:

> - Added example and documentation for FIPS compliant communication with Kafka cluster

Reviewer: ```suggestion
- Added documentation with an example of FIPS compliant communication with Kafka cluster.
```

Author: Used suggestion
